plugins/GSdx/GS.h

```diff
@@ -1414,3 +1414,13 @@ enum class GSVideoMode : uint8
 	DTV_720P,
 	DTV_1080I
 };
+
```

Comment:
```
+	Trilinear_Forced,
+	Trilinear_Slow
```
Hum, Triliner_Slow => Trilinear_Always ? Trilinear_Forced => Triliner_Bilinear_Forced ? I'm sure we can find better name.